### PR TITLE
[NFC] Use strict comparison where  possible

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Minor code standards cleanup

Before
----------------------------------------
Non-strict comparison used when comparing against strings

After
----------------------------------------
Strict comparison used

Technical Details
----------------------------------------
Also fixes a couple of comments & uses of Array::value

Comments
----------------------------------------
Strict comparison should always be used where possible

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
test this please

File: CRM/Activity/Form/Activity.php [hunk 1]
```diff
@@ -823,7 +823,7 @@ public function buildQuickForm() {
    */
   public static function formRule($fields, $files, $self) {
     // skip form rule if deleting
-    if (CRM_Utils_Array::value('_qf_Activity_next_', $fields) == 'Delete') {
+    if ($fields['_qf_Activity_next_'] ?? NULL === 'Delete') {
```
Comment (colemanw) [inline CRM/Activity/Form/Activity.php]: This seems a little ambiguous about which operator goes first (?? or ===)
```suggestion
    if (($fields['_qf_Activity_next_'] ?? NULL) === 'Delete') {
```

File: CRM/Activity/Form/Activity.php [hunk 2]
```diff
@@ -958,7 +958,7 @@ public function postProcess($params = NULL) {
       $this->_activityId
     );
 
-    $params['is_multi_activity'] = CRM_Utils_Array::value('separation', $params) == 'separate';
+    $params['is_multi_activity'] = $params['separation'] ?? NULL === 'separate';
```
Comment (colemanw) [inline CRM/Activity/Form/Activity.php]:
```suggestion
    $params['is_multi_activity'] = ($params['separation'] ?? NULL) === 'separate';
```

File: CRM/Activity/Form/Activity.php (hunk 1 above)
Comment (eileenmcnaughton) [reply CRM/Activity/Form/Activity.php]: lol -  I was going the brackets but  thought  you  have been removing them from lots of places where I wanted them

File: CRM/Activity/Form/Activity.php (hunk 2 above)
Comment (eileenmcnaughton) [reply CRM/Activity/Form/Activity.php]: @colemanw fixed

**Comment by colemanw** [comment]
Unrelated fail.

**Pull request merged**